gemfire-shared/src/main/java/com/gemstone/gemfire/internal/cache/Status.java

```diff
@@ -296,4 +296,23 @@ public String shortStatus() {
     }
     return buffer.toString();
   }
+
+  public static String getStatus(int state) {
```

Comment:
Maybe rename to "getStateString". Also making it a member rather than static will be cleaner. Also change the code in shortStatus to use this method instead of its own switch case.

Reply:
ok.